core/pta/device.c

```diff
@@ -21,18 +21,20 @@
 #define PTA_NAME "device.pta"
 
 static void add_ta(uint32_t flags, const TEE_UUID *uuid, uint8_t *buf,
-		   uint32_t blen, uint32_t *pos)
+		   uint32_t blen, uint32_t *pos, uint32_t rflags)
 {
-	if (flags & TA_FLAG_DEVICE_ENUM) {
-		if (*pos + sizeof(*uuid) <= blen)
-			tee_uuid_to_octets(buf + *pos, uuid);
+	if (!(flags & rflags & TA_FLAG_DEVICE_ENUM_MASK))
```

Comment:
`TA_FLAG_DEVICE_ENUM_MASK` doesn't seem very useful to me.

Reply:
I think rflags can have more additional  flags in future and it's better to write here that we are looking only for 2 bits.

Reply:
Current there's two callers, one with `TA_FLAG_DEVICE_ENUM` and another with `TA_FLAG_DEVICE_ENUM_SUPP`. What will you do if you add another bit? Probably expanding the mask in order to allow that other bit. To me it seems pretty useless, at least for now.